.github/workflows/main.yml

```diff
@@ -34,9 +34,11 @@ jobs:
       matrix:
         include:
           - dependencies: black python3-isort
-            task: fmt-travis
+            task: make -f Makefile fmt-travis
           - dependencies: yamllint
-            task: yamllint
+            task: make -f Makefile yamllint
+          - dependencies: pylint python3-pyparsing python3-hs-dbus-signature
```

Comment:
Looks in the project's setup.py and note that it has no depedencies at all. So you can drop python3-pyparsing. _But_ in the tests it does use python3-hs-dbus-signature, so you'll have to keep that.